agent/api/container/container.go

```diff
@@ -190,6 +190,9 @@ type Container struct {
 	// KnownPortBindingsUnsafe is an array of port bindings for the container.
 	KnownPortBindingsUnsafe []PortBinding `json:"KnownPortBindings"`
 
+	// VolumesUnsafe is an array of volume mounts in the container.
+	VolumesUnsafe []docker.Mount `json:"-"`
```

Comment:
Please update this unit test to cover this: https://github.com/aws/amazon-ecs-agent/blob/02a6a9fff9bee5e0bbd2b936d4ceee50207595f8/agent/engine/docker_task_engine_test.go#L1923